### DISCUSSION TITLE
Is There A Way To View The Log / Console Output On Mobile?

### DISCUSSION THREAD

**Discussion by InfiniteEchoDev** [Vinzent03/obsidian-git#754] [Q&A]
First off, thanks for the great plugin.  It works well.  Also, I'm having an issue where the Push is failing (hanging forever) on my mobile device and am collecting information about the issue before posting about it in Issues.  However, on mobile there doesn't seem to be a way to open the Inspector (ctrl + shift + i on desktop) so I can't see the console output logging at all.  Unless... 

Is there a way to see the console output / runtime log on a mobile device?

**Comment by Vinzent03** [comment]
To be honest, I don't think any more insight on hanging pushing will help, but the [documentation here](https://docs.obsidian.md/Plugins/Getting+started/Mobile+development#Inspecting+the+webview+on+the+actual+mobile+device) should help you.

**Comment by MathCrafted** [comment]
You can also use logcat in Android Studio.
